hera_xblocks/hera_pages/hera_pages/static/html/hera_pages.html

```diff
@@ -2,22 +2,94 @@
     <div class="author-block__content">
         <div class="author-block__image">
             <div class="image-wrapper">
-                {% if data.imgUrl %}
-                {% for img in data.imgUrl %}
-                <img src="{{ img }}" alt=""/>
-                {% endfor %}
-                {% elif data.iframeUrl %}
-                <iframe src="{{ data.iframeUrl }}"></iframe>
-                {% else %}
-                <img alt="Unfortunately no image"/>
-                {% endif %}
+                % if data.get('imgUrl'):
+                    % for img in data['imgUrl']:
+                        <img src="${ img }" alt=""/>
+                    % endfor
+                % elif data.get('iframeUrl'):
+                    <iframe src="${ data['iframeUrl'] }"></iframe>
+                % else:
+                    <img alt="Unfortunately no image"/>
+                % endif
             </div>
         </div>
         <div class="author-block__question">
             <div class="slidebar-wrapper">
-                {% for i in  data.sliderBar %}
-                <div class="single-item">{{ i.content|safe }}</div>
-                {% endfor %}
+                % if data.get('sliderBar'):
+                    % for index, item in enumerate(data['sliderBar']):
+                        <div class="single-item">
+                            <div>
+                                ${ item['content'] }
+                            </div>
+                            % if item.get('tableData'):
+                                <div class="table-block">
+                                    <% user_answer_input_index = 0 %>
+                                    <form class="table-form-${index}">
+                                        <table>
+                                            <tbody>
+                                                % if item['tableData'].get('columns'):
+                                                    <tr>
+                                                        % for column in item['tableData']['columns']:
+                                                            % if column['type'] == 'head':
```

Comment:
```suggestion
                                                            % if column.get('type') == 'head':
```

Reply:
not all cells have this key (as appeared, need to be fixed in cms part)

Reply:
Done